gef.py

```diff
@@ -1433,17 +1433,17 @@ def is_conditional_branch(self, insn):
         return insn.mnemonic.startswith("b")
 
     def is_branch_taken(self, insn):
-        def convert_to_signed(x):
+        def long_to_twos_compliment(v):
             """Convert a python long value to its two's compliment"""
             if is_elf32():
-                if (x & 0x80000000) != 0:
-                    return x - 0x100000000
+                if v & 0x80000000 != 0:
```

Comment:
you don't need the != 0, just `if v & ...`

Reply:
True